mathesar_ui/src/component-library/tutorial/Tutorial.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
```

Comment:
Will create separate issues for writing stories for Alert and Tutorial component once the PR is merged

Reply:
I wouldn't bother spending any time on that. I sometimes write stories for components because in certain situations speeds up the process of building lower-level components and testing their various scenarios in isolation. But writing the stories for the sake of documentation is something to defer until well after our first release I'd say.